code/AEPSDK.brs

```diff
@@ -203,24 +203,24 @@ function AdobeAEPSDKInit(taskNode = invalid as dynamic) as object
         '
         ' This function allows passing custom identifiers using identityMap.
         '
-        ' @param data as object : xdm data
+        ' @param data as object : Dictionary containg xdm and freeform data
         ' @param [optional] callback as function(context, result) : handle Edge response
         ' @param [optional] context as dynamic : context to be passed to the callback function
         '
         ' *************************************************************************************
 
-        sendEvent: function(xdmData as object, callback = _adb_defaultCallback as function, context = invalid as dynamic) as void
+        sendEvent: function(data as object, callback = _adb_defaultCallback as function, context = invalid as dynamic) as void
             _adb_logDebug("API: sendEvent()")
-            if _adb_isEmptyOrInvalidMap(xdmData) then
-                _adb_logError("sendEvent() - Cannot send event, invalid XDM data")
+
+            if _adb_isEmptyOrInvalidMap(data.xdm) then
+                _adb_logError("sendEvent() - Cannot send event, XDM data not found.")
                 return
```

Comment:
since this api now supports both xdm and data, can you confirm this check is a requirement from PM perspective. if so maybe we can validate the event has either xdm or data before processing the request

Reply:
As per our discussion on slack, we are tending to keep xdm as required, awaiting PM approval.